src/damage.cpp

```diff
@@ -190,7 +190,11 @@ void ammo_effects( const tripoint &p, const std::set<std::string> &effects )
             g->m.add_field( pt, fd_tear_gas, 3, 0 );
         }
     }
-
+    if( effects.count( "GAS_FUNGICIDAL" ) > 0 ) {
+        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
+            g->m.add_field( pt, fd_fungicidal_gas, 4, 0 );
```

Comment:
I thought there was some kind of counter that reduces field density after a certain amount of time. Figured, if I set it higher the gas remains there longer, thereby simulating the gas 'ganing in the air' after it has been sprayed.
I'll look into spread_gas, that sounds exactly like what I need to modify.